sunpy/map/tests/test_reproject_to.py

```diff
@@ -236,3 +222,41 @@ def test_reproject_to_auto_extent_wcs(aia171_test_map, auto_extent, crpix, pixel
     # These parts of the WCS should have changed
     assert_quantity_allclose(reprojected_wcs.wcs.crpix, crpix)
     assert_quantity_allclose(reprojected_wcs.pixel_shape, pixel_shape)
+
+
+@figure_test
+@pytest.mark.parametrize("screen", [sunpy.coordinates.SphericalScreen, sunpy.coordinates.PlanarScreen])
+def test_reproject_to_screen_plus_diffrot(aia171_test_map, screen):
+    new_time = aia171_test_map.date + 5*u.day
+    header = {
+        'naxis1': 150,
+        'naxis2': 150,
+        'crpix1': 75.5,
+        'crpix2': 75.5,
+        'crval1': 0,
+        'crval2': 0,
+        'cdelt1': 20,
+        'cdelt2': 20,
+        'cunit1': 'arcsec',
+        'cunit2': 'arcsec',
+        'ctype1': 'HPLN-TAN',
+        'ctype2': 'HPLT-TAN',
+        'hgln_obs': 0,
+        'hglt_obs': 30,
+        'rsun_ref': aia171_test_map.rsun_meters.value,
+        'dsun_obs': aia171_test_map.dsun.value,
+        'date-obs': new_time.isot,
+        'mjd-obs': new_time.mjd,
+    }
```

Comment:
Why not just use the header helper here?

Reply:
Certainly should for any public example, but for unit tests, I tend to avoid using the header helper when a simple header can be constructed by hand in order to minimize the "blast radius" of test failures if a bug crops up in the header helper